### DISCUSSION TITLE
Why does this look like a copy of Discord and not something original?

### DISCUSSION THREAD

**Discussion by RutsuKun** [stoatchat/discussions#396] [General]
same as above

**Comment by Zomatree** [comment]
Why does Discord look like a copy of Slack and not something original?

**Comment by Rexogamer** [comment]
Discord's design is based on Slack's - in addition, we natively allow you to use custom themes and CSS in the official client (Revite) and the official instance permits the use of custom clients.

**Comment by RutsuKun** [comment]
Discord doesn't look like 1:1 Slack, they only took inspiration. But Revolt looks like a 1:1 copy of Discord.

**Reply by Rexogamer** [reply]
I'd argue that, side by side, you'd easily be able to tell them apart; and again, you can natively customise the client to make it look however you want. Is there anything in particular you want to see?

**Comment by 4444dogs** [comment]
Revolt doesn't exist to be different from Discord. It exists to please the privacy community while still maintaining a familiar layout.
